server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyToResponseBody.scala

```diff
@@ -0,0 +1,52 @@
+package sttp.tapir.server.netty
+
+import java.io.InputStream
+import java.nio.ByteBuffer
+import java.nio.charset.Charset
+
+import io.netty.buffer.{ByteBuf, Unpooled}
+import sttp.capabilities
+import sttp.model.HasHeaders
+import sttp.tapir.{CodecFormat, RawBodyType, WebSocketBodyOutput}
+import sttp.tapir.internal.NoStreams
+import sttp.tapir.server.interpreter.ToResponseBody
+
+class NettyToResponseBody extends ToResponseBody[ByteBuf, NoStreams] {
+  override val streams: capabilities.Streams[NoStreams] = NoStreams
+
+  override def fromRawValue[R](v: R, headers: HasHeaders, format: CodecFormat, bodyType: RawBodyType[R]): ByteBuf = {
+    bodyType match {
+      case RawBodyType.StringBody(charset) =>
+        Unpooled.copiedBuffer(v.toString, charset)
+      case RawBodyType.ByteArrayBody =>
+        val bytes = v.asInstanceOf[Array[Byte]]
+        Unpooled.copiedBuffer(bytes)
+      case RawBodyType.ByteBufferBody =>
+        val byteBuffer = v.asInstanceOf[ByteBuffer]
+        Unpooled.copiedBuffer(byteBuffer)
+
+      case RawBodyType.InputStreamBody =>
+        val stream = v.asInstanceOf[InputStream]
+
+        val buf = Unpooled.buffer()
+        //todo
+        buf.writeBytes(stream, 6000)
```

Comment:
since for now we are working with non-streaming, we need to read the whole InputStream into memory and create a ByteBuf from it